app/src/main/java/net/skyscanner/backpack/demo/data/ExampleBpkCalendarController.kt

```diff
@@ -74,9 +74,9 @@ internal fun multiColoredExampleCalendarColoring(
   }
   return CalendarColoring(
     setOf(
-      ColoredBucket(ContextCompat.getColor(context, R.color.bpkRed200), redSet),
-      ColoredBucket(ContextCompat.getColor(context, R.color.bpkYellow100), yellowSet),
-      ColoredBucket(ContextCompat.getColor(context, R.color.bpkGreen100), greenSet),
+      ColoredBucket(ContextCompat.getColor(context, R.color.bpkNara), redSet),
```

Comment:
Let's use azureMonteverde, azureKolkata, azurePanjin here.

Reply:
those are not the right colours see https://app.zeplin.io/project/5d6fa72570db2f18bdf8e209/screen/5d6fb58fc1b83b981a512b5e